luthier/config/src/dimensions.lua

```diff
@@ -121,209 +122,218 @@ local searchProviders = dimensionUtils.searchProviders
 local keyValueStores = dimensionUtils.keyValueStores
 
 return {
+    testDimension = {
+        longName = "a longName for testing",
+        description = "a description for testing",
+        fields = FIELDSETS.test,
+        category = "a category for testing",
+        type = "KeyValueStoreDimensionFactory",
```

Comment:
Personally, I would call this `KeyValueStoreDimension`, rather than `KeyValueStoreDimensionFactory`. Yes,
we're using the `KeyValueStoreDimensionFactory` to build a `KeyValueStoreDimension`, and we're using this
as a key into a map of factories, but customers don't care
about that. They only care about what kind of dimension this is, and it's a `KeyValueStoreDimension`.

Reply:
Changed.